gzipi/lib.py

```diff
@@ -426,8 +576,14 @@ def _is_last_line(fin, start, end):
         return False
 
 
-def _binary_search(key, fin, fsize, delimiter=b'|', lineterminator=b'\n',
-                   buffer_size=_DEFAULT_BUFFER_SIZE):
+def _binary_search(
+    key: bytes,
+    fin: IO[bytes],
+    fsize: int,
+    delimiter: bytes = b'|',
```

Comment:
Lets move delimiter, lineterminator and any other format parameters into constants at the top of the submodule.